internal/status/glob.go

```diff
@@ -26,13 +25,13 @@ func globs(dir string, globs []string) ([]string, error) {
 func Glob(dir string, g string) ([]string, error) {
 	files := make([]string, 0)
 	g = filepathext.SmartJoin(dir, g)
-
 	g, err := execext.Expand(g)
 	if err != nil {
 		return nil, err
 	}
 
-	fs, err := zglob.Glob(g)
+	//fs, err := zglob.Glob(g) //Looks like this does not evaluate **/* glob correctly
+	fs, err := filepath.Glob(g)
```

Comment:
`filepath.Glob` does not support double star (`foo/**/*.js`). We should keep using `zglob.Glob`

Reply:
Ok, Thanks I will dig more.
Zglob does not follow symlinks so I will try to figure out how to have both ** and symlinks
https://github.com/mattn/go-zglob/issues/3